### DISCUSSION TITLE
GET availablepackages "Not supported yet"

### DISCUSSION THREAD

**Discussion by xingyunyang01** [kubeapps/kubeapps#5790] [Q&A]
When I called API "GET  http://172.32.1.21:32666/apis/plugins/helm/packages/v1alpha1/availablepackages", it response 
{
    "code": 12,
    "message": "Not supported yet: not including a namespace means that it returns everything a user can read"
}

I don't know how do I need to call

**Comment by absoludity** [comment]
If you're using the REST API, then yes, you'll need to provide the namespace as a query param there, such as "?context.namespace=default" .

This is described in the swagger documentation available in-app.
